cost-model/src/transaction_cost.rs

```diff
@@ -67,11 +67,18 @@ impl TransactionCost {
     }
 
     pub fn sum(&self) -> u64 {
```

Comment:
nit: could be more readable as:
```rust
let mut cost = self.signature_cost
               .saturating_add(self.write_lock_cost)
               .saturating_add(self.data_bytes_cost)
               .saturating_add(self.builtins_execution_cost);
if !self.is_simple_vote {
    cost = cost.saturating_add(self.bpf_execution_cost)
              .saturating_add(self.loaded_accounts_data_size_cost)
}
cost
```

Reply:
I have a follow-up PR #33230 to separate vote cost from transaction's. Thought to keep the bug-fixing pr simple, and easy to remove code for vote-path